### ISSUE TITLE
Accessibility/Authentication: Unable to move the required actions in the auth settings using the keyboard 

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#18102]
### Describe the bug

The user can't move the required actions in the auth settings using the keyboard. This should work the same as the Flows which are accessible via tabs + arrows.

Tested on Windows 10, build 19044.1889, Firefox 103.0.2 or Microsoft Edge 104.0.1293.54

Note: I have also checked this issue and it's present in Chrome.

### Version

19.0.0

### Expected behavior

The user can move the required actions in the auth settings using the keyboard.

### Actual behavior

Nothing happens when trying to move the selected action using tabs/arrows.

### How to Reproduce?

- Go to Authentication from the left-side menu
- Navigate through tabs and verify that all operations are performant and information readable
